pennylane/templates/state_preparations/basis.py

```diff
@@ -99,6 +99,7 @@ def compute_decomposition(basis_state, wires):  # pylint: disable=arguments-diff
         Args:
             basis_state (array): Input array of shape ``(len(wires),)``
             wires (Any or Iterable[Any]): wires that the operator acts on
+            batch_size (Optional[int]): Size of batch, if any
```

Comment:
Why is this argument getting added?

Reply:
Sorry, I added it before before realizing I didn't need it. Forgot to remove the docstring. Fixed.